homework/App.js

```diff
@@ -17,18 +17,40 @@ class App {
     // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
     const root = document.getElementById('root');
+    Util.createAndAppend('div', root, { id: 'container' });
 
     Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
 
     try {
       const repos = await Util.fetchJSON(url);
       this.repos = repos.map(repo => new Repository(repo));
-      // TODO: add your own code here
+
+      const select = this.generateSelections(repos, root);
+      this.fetchContributorsAndRender(repos.indexOf(this.repos[0].repository));
```

Comment:
All this line is just about select the 0 index from the repos array, so istead you may do the following:
```suggestion
      this.fetchContributorsAndRender(0); // it's same result as yours 👆
```
because you have:
  - `this.repos[0].repository` is the repositories array
  - `repos.indexOf` with it call will give you the index of your item which is as you spicified the index number `[0]`.

So all together going to give you as a result:
```javascript
      console.log(repos.indexOf(this.repos[0].repository)) // 0
```